src/main.js

```diff
@@ -62,6 +62,23 @@ function getUserById(userId) {
     return user;
 }
 
+function copyListToClipboard() {
+    const sortedList = [...nonFollowersList].sort((a, b) => (a.username > b.username ? 1 : -1));
+
+    let output = '';
+    sortedList.forEach(user => {
+        output += user.username + '\n';
+    });
+
+    copyToClipboard(output);
+}
+
+function copyToClipboard(text) {
+    navigator.clipboard.writeText(text);
```

Comment:
Missing await. This can lead in a DOM not focus exception. Because the alert will pop before copying the text